Open-Science-Workshop-2022/tutorials/Scientific_Libraries_Numpy.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<p>One other small comment - can you make the title here something like "Working with NumPy" to be consistent with the other two notebooks? This is the tile that will show up in the side bar for the jupyterbook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-Development/ARM-Notebooks/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>